### PR TITLE
CLDC-1847 Add person 6 for non joint purchases

### PR DESCRIPTION
**Pull request by kosiakkatrina**
Update household count hint and validation to allow 5 extra people instead of 4 if it is not a joint purchase
Add missing person 6 pages

### CODE REVIEW

File: spec/models/form/sales/questions/number_of_others_in_property_spec.rb [hunk 1]
```diff
@@ -1,11 +1,12 @@
 require "rails_helper"
 
 RSpec.describe Form::Sales::Questions::NumberOfOthersInProperty, type: :model do
-  subject(:question) { described_class.new(question_id, question_definition, page) }
+  subject(:question) { described_class.new(question_id, question_definition, page, joint_purchase:) }
 
   let(:question_id) { nil }
   let(:question_definition) { nil }
   let(:page) { instance_double(Form::Page) }
+  let(:joint_purchase) { true }
```
Comment (natdeanlewissoftwire) [inline spec/models/form/sales/questions/number_of_others_in_property_spec.rb:9]: should we also test when joint_purchase == false?

File: app/models/form/sales/pages/number_of_others_in_property.rb [hunk 2]
```diff
@@ -5,16 +5,18 @@ def initialize(id, hsh, subsection)
...
         "noint" => 1,
+        "jointpur" => 2,
       },
     ]
   end
 
   def questions
     @questions ||= [
-      Form::Sales::Questions::NumberOfOthersInProperty.new(nil, nil, self),
+      Form::Sales::Questions::NumberOfOthersInProperty.new(nil, nil, self, joint_purchase: false),
```
Comment (natdeanlewissoftwire) [inline app/models/form/sales/pages/number_of_others_in_property.rb]: can we set joint_purchase dynamically and remove the nede for number_of_others_in_property_joint_purchase.rb? Or is that more effort than it's worth?

**Review by natdeanlewissoftwire — APPROVED** [review]
lgtm, couple of queries :)

File: spec/models/form/sales/questions/number_of_others_in_property_spec.rb (hunk 1 above)
Comment (kosiakkatrina) [reply spec/models/form/sales/questions/number_of_others_in_property_spec.rb:9]: There are a few tests that check the non joint purchase fields on line 47, but only the fields that are different

File: app/models/form/sales/pages/number_of_others_in_property.rb (hunk 2 above)
Comment (kosiakkatrina) [reply app/models/form/sales/pages/number_of_others_in_property.rb]: Yeah, probably, have a look now?

File: app/models/form/sales/pages/number_of_others_in_property.rb [hunk 3]
```diff
@@ -1,20 +1,22 @@
...
+  def initialize(id, hsh, subsection, joint_purchase:)
+    super(id, hsh, subsection)
     @depends_on = [
       {
         "privacynotice" => 1,
+        "jointpur" => joint_purchase ? 1 : 2,
       },
       {
         "noint" => 1,
+        "jointpur" => joint_purchase ? 1 : 2,
```
Comment (arfacamble) [inline app/models/form/sales/pages/number_of_others_in_property.rb:11]:
```suggestion
        "joint_purchase?" => joint_purchase,
```

**Pull request merged**